### DISCUSSION TITLE
ios剪贴板发送至folomo带标签版（基于send to folomo 与 copy 2 folomo 修改）

### DISCUSSION THREAD

**Discussion by zooges** [flomoapp/3rd-party-tools#27] [分享创造]
https://www.icloud.com/shortcuts/fdd319d5bc544f96b4f52a523c4fcced

https://user-images.githubusercontent.com/65641894/107248700-8331f180-6a6d-11eb-955c-2c591718683c.MP4

**Comment by fsafaf** [comment]
有个问题:剪贴板是否为空，都会触发 剪贴板为空 的提醒
